src/app/pet-form/pet-form.component.html

```diff
@@ -1,108 +1,105 @@
+          <mat-hint>MM/DD/YYYY</mat-hint>
+          <mat-datepicker-toggle matSuffix [for]="birthday"></mat-datepicker-toggle>
+          <mat-datepicker #birthday></mat-datepicker>
+        </mat-form-field>
+      </p>
+      <p>
+        <mat-form-field appearance="outline">
+          <mat-label>Adoption Day</mat-label>
+          <input matInput [matDatepicker]="adoptionDay" formControlName="adoptionDay">
+          <mat-hint>MM/DD/YYYY</mat-hint>
+          <mat-datepicker-toggle matSuffix [for]="adoptionDay"></mat-datepicker-toggle>
+          <mat-datepicker #adoptionDay></mat-datepicker>
+        </mat-form-field>
+      </p>
+      <p class="bottom-margin">
+        <mat-radio-group aria-label="Select your pet's sex" formControlName="sex" required>
+          <mat-label [ngClass]="{validate: this.newPetForm.get('sex')?.invalid && this.submitted}">Sex*</mat-label>
+          <mat-radio-button id="male" name="sex" value="Male">Male</mat-radio-button>
+          <mat-radio-button id="female" name="sex" value="Female">Female</mat-radio-button>
+          <mat-icon class="center-icon" *ngIf="this.newPetForm.get('sex')?.invalid && this.submitted" color="warn">
+            cancel
+          </mat-icon>
+        </mat-radio-group>
+      </p>
+      <p class="bottom-margin">
+        <mat-checkbox formControlName="fixed" id="fixed">Spayed/Neutered?</mat-checkbox>
+      </p>
+      <p class="bottom-margin">
+        <mat-label>Vet Records </mat-label>
+        <button type="button" mat-mini-fab color="primary" (click)="vetRecords.click()">
+          <mat-icon *ngIf="!this.vetRecordsFile">attach_file</mat-icon>
+          <mat-icon *ngIf="this.vetRecordsFile">check_circle</mat-icon>
+        </button>
+        <mat-label class="file-label" *ngIf="this.vetRecordsFile">{{this.vetRecordsFile.name}}</mat-label>
+        <input (change)="onChangeVetRecords($event)" hidden formControlName="vetRecords" #vetRecords type="file"
+          accept=".pdf">
+      </p>
+      <p>
+        <mat-label>Pet Photo</mat-label>
+        <button type="button" mat-mini-fab color="primary" (click)="petPhoto.click()">
+          <mat-icon *ngIf="!this.petPhotoFile">attach_file</mat-icon>
+          <mat-icon *ngIf="this.petPhotoFile">check_circle</mat-icon>
+        </button>
+        <mat-label class="file-label" *ngIf="this.petPhotoFile">{{this.petPhotoFile.name}}</mat-label>
+        <input (change)="onChangePetPhoto($event)" hidden formControlName="petPhoto" #petPhoto type="file"
+          accept="image/*">
+      </p>
+    </div>
     <h3 class="inline">Prescriptions</h3>
     <button mat-mini-fab color="primary" aria-label="icon button with a plus sign to add new prescription"
       class="inline" (click)="addPrescription()" type="button">
       <mat-icon>add_icon</mat-icon>
     </button>
-    <div formArrayName="prescriptions">
+    <div formArrayName="prescriptions" class="border-bottom">
```

Comment:
Remember to name classes based on the functions they serve and not the styles they represent. For example you might have a "card" style or a "prescription-card" but wouldn't have a "add-margin" or "extra-space" class.

Reply:
I believe that I corrected this issue in the new commit. If you still see room for improvement, please do let me know.